openfisca_france/param/param.xml

```diff
@@ -2781,7 +2784,16 @@
         <VALUE deb="2010-01-01" fin="2011-12-31" valeur="878" />
         <VALUE deb="2012-01-01" fin="2012-12-31" valeur="960" />
         <VALUE deb="2013-01-01" fin="2013-12-31" valeur="1016" />
-        <VALUE deb="2014-01-01" fin="2014-12-31" valeur="1021" />
```

Comment:
@iDevelopper thanx. If you just fill 1AJ (which is net imposable on your fiche de paye) you have to substract 10% as an rebate on your business expenses. So most people will have to put 16666 (15000*0.9) euros to get your exemple. Feel free to implement it in a .yaml test and commit it :-) !